src/ansys/fluent/core/session.py

```diff
@@ -258,75 +272,63 @@ def __exit__(self, exc_type: Any, exc_val: Any, exc_tb: Any):
         logger.debug("session.__exit__() called")
         self.exit()
 
-    def upload(self, file_name: str, remote_file_name: Optional[str] = None):
-        """Uploads a file on the server."""
-        if not self._uploader:
-            self._uploader = _Uploader(self.fluent_connection._remote_instance)
-        return self._uploader.upload(file_name, remote_file_name)
-
-    def download(self, file_name: str, local_file_name: Optional[str] = None):
-        """Downloads a file from the server."""
-        if not self._uploader:
-            self._uploader = _Uploader(self.fluent_connection._remote_instance)
-        return self._uploader.download(file_name, local_file_name)
-
-
-class _Uploader:
-    """Instantiates a file uploader and downloader to have a seamless file reading /
-    writing in the cloud particularly in Ansys lab . Here we are exposing upload and
-    download methods on session objects. These would be no- ops if PyPIM is not
-    configured or not authorized with the appropriate service. This will be used for
-    internal purpose only.
-
-    Attributes
-    ----------
-    pim_instance: PIM instance
-        Instance of PIM which supports upload server services.
-
-    file_service: Client instance
-        Instance of Client which supports upload and download methods.
-
-    Methods
-    -------
-    upload(
-        file_name, remote_file_name
-        )
-        Upload a file to the server.
-
-    download(
-        file_name, local_file_name
-        )
-        Download a file from the server.
-    """
-
-    def __init__(self, pim_instance):
-        self.pim_instance = pim_instance
-        self.file_service = None
-        try:
-            upload_server = self.pim_instance.services["http-simple-upload-server"]
-        except (AttributeError, KeyError):
-            pass
-        else:
-            from simple_upload_server.client import Client
+    def _upload_download_helper(
```

Comment:
@hpohekar 
There are too many responsibilities being managed in this module.

Rather than merging the uploader code with new code, it would be better to keep them separate. The Uploader doesn't even need to be in this module. It could have been retained as it was and moved into a new module. Then your new functionality could have been built on that - separately. This session class would simply be a user of that.

Reply:
Ok. We can create a separate file `uploader.py` will move `_Uploader` class to it.

Reply:
Done. Thank you.